com.obsproject.Studio.Plugin.RewardsTheater.yaml

```diff
@@ -0,0 +1,57 @@
+id: com.obsproject.Studio.Plugin.RewardsTheater
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
```

Comment:
This is EOL. OBS is using 6.5 https://github.com/obsproject/obs-studio/blob/35f84814986d659d1c9416d7de09973c3f5cdc15/build-aux/com.obsproject.Studio.json#L4